tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/ClientEncryptionProseTests.cs

```diff
@@ -1522,38 +1525,45 @@ void KmsProviderEndpointConfigurator(string kmsProviderName, Dictionary<string,
             }
         }
 
+        [Trait("Category", "CsfleGCPKMS")]
```

Comment:
This test is for both `aws` and `gcp`, but only categorized as `CsfleGCPKMS`. Do we have a similar AWS-related category? If not, why not?

Reply:
this says that additionally to all CSFLE envs we also should launch this particular test in GCP. We don't run this on aws env (which will require the same Trait) because spec explicitly says to work only with env variable (which handled inside test). So in other words, we should validate behavior at least with one configuration, aws allows to do it with env variable, this is why we use this (simplest approach), but gcp doesn't allow doing it, then we have no choice other that configuring gcp env for this test